superset/models/dashboard.py

```diff
@@ -242,29 +242,24 @@ def data(self) -> Dict[str, Any]:
 
     @cache_manager.cache.memoize(
         # manage cache version manually
-        make_name=lambda fname: f"{fname}-v2.1",
+        make_name=lambda fname: f"{fname}-v1.0",
         unless=lambda: not is_feature_enabled("DASHBOARD_CACHE"),
     )
-    def full_data(self) -> Dict[str, Any]:
-        """Bootstrap data for rendering the dashboard page."""
-        slices = self.slices
-        datasource_slices = utils.indexed(slices, "datasource")
-        try:
-            datasources = {
-                # Filter out unneeded fields from the datasource payload
-                datasource.uid: datasource.data_for_slices(slices)
-                for datasource, slices in datasource_slices.items()
-                if datasource
-            }
-        except (SupersetException, SQLAlchemyError):
-            datasources = {}
+    def slices_cached(self) -> List[Slice]:
```

Comment:
instead of naming this with the `_cached` suffix, can we just make this a `@property` function named `slices`? Then it'll always be used everywhere, but it'll only be cached if the feature flag is enabled

Reply:
As discussed in slack, changing `slices` to a property ends up being kinda sketch so leaving it like this for now.